extensions/wikia/EditPreview/EditPreview.i18n.php

```diff
@@ -3,28 +3,32 @@
 
 /** English */
 $messages['en'] = array(
-	'wikia-editor-preview-desc' => 'Enables users to preview their edits before saving them',
-	'wikia-editor-preview-current-width' => 'Desktop',
-	'wikia-editor-preview-min-width' => 'Tablet Portrait',
-	'wikia-editor-preview-max-width' => 'Desktop XL',
-	'wikia-editor-preview-mobile-width' => 'Mobile',
-	'wikia-editor-preview-venus-width' => 'Venus',
-	'wikia-editor-preview-type-tooltip' => "Changing this option will show you what this article will look like when it's displayed in a browser on small screens, large screens, mobile devices, or your current display.",
 	'wikia-editor-preview-best-practices-notice' => 'Did you know that over 50% of Wikia visitors are reading your content on mobile? Some custom templates may not properly show on mobile devices. To make sure everything looks great to all visitors, follow our best practices.',
 	'wikia-editor-preview-best-practices-button' => 'Best practices',
 	'wikia-editor-preview-best-practices-button-link' => 'http://community.wikia.com/wiki/Help:Wikitext_best_practices',
+	'wikia-editor-preview-current-width' => 'Desktop',
+	'wikia-editor-preview-desc' => 'Enables users to preview their edits before saving them',
+	'wikia-editor-preview-max-width' => 'Desktop XL',
+	'wikia-editor-preview-min-width' => 'Tablet Portrait',
+	'wikia-editor-preview-mercury' => 'Mobile (Mercury)',
```

Comment:
Mercury is not a 'public' name for new skin - can you talk with some PO on how to name it here?